data/Ousman37.json

```
@@ -1,17 +1,87 @@
 {
   "name": "Ousman Ethmane",
   "bio": "Passionate Front-End Developer | UI/UX Enthusiast | Building innovative web experiences",
+  "avatar": "https://github.com/Ousman37.png",
```

this is also not necessary: 
```suggestion
  "bio": "Passionate Front-End Developer | UI/UX Enthusiast | Building innovative web experiences",
```